src/components/nav/NavBar.vue

```diff
@@ -1,14 +1,60 @@
 <template>
   <div id="nav">
-    <router-link to="/" class="nav-home-link">
-      <img src="../../assets/c4c-banner-logo.png" class="home-nav" />
+    <div class="big-menu">
+      <router-link to="/" class="nav-home-link">
+        <img src="../../assets/c4c-banner-logo.png" class="home-nav" />
+      </router-link>
+      <router-link to="/news" class="nav-item">News</router-link>
+      <router-link to="/events" class="nav-item">Events</router-link>
+      <router-link to="/board" class="nav-item">Board</router-link>
+      <router-link to="/apply" class="nav-item">Apply</router-link>
+      <router-link to="/auth" class="nav-item">Sign In</router-link>
+    </div>
+    <img
```

Comment:
Could we pin this accordion icon to the far right corner of the navbar? currently it's appearing without padding on the left of the logo image.